src/Application.cpp

```diff
@@ -76,7 +78,11 @@ Application::Application(Settings &_settings, Paths &_paths)
     , chatterinoBadges(&this->emplace<ChatterinoBadges>())
     , ffzBadges(&this->emplace<FfzBadges>())
     , seventvBadges(&this->emplace<SeventvBadges>())
+<<<<<<< HEAD
     , seventvPaints(&this->emplace<SeventvPaints>())
+=======
+>>>>>>> upstream/master
```

Comment:
```suggestion
    , seventvPaints(&this->emplace<SeventvPaints>())
```

Reply:
i was using this pr to stash changes while i redid the merge.